packages/syft/src/syft/service/api/api.py

```diff
@@ -117,6 +117,11 @@ class Endpoint(SyftObject):
     state: dict[Any, Any] | None = None
     signature: Signature
 
+    __exclude_sync_diff_attrs__ = ["state"]
+
+    def __repr__(self) -> str:
```

Comment:
why do we do this? This means in every context we only see the code, which may be confusing

Reply:
we could either make a specialised function for representing the object in this context or update the __repr__ to be more genericly usable (with other properties added, that at least make clear its a class)